subprojects/test-runner/report-api/src/main/kotlin/com/avito/report/serialize/ReportSerializer.kt

```diff
@@ -14,9 +14,9 @@ public interface ReportSerializer {
     public fun deserialize(reportFile: File): Result<TestRuntimeData>
 }
 
-public fun ReportSerializer(): ReportSerializer = GsonReportSerializer(gson = reportGson)
+public fun ReportSerializer(): ReportSerializer = GsonReportSerializer(gson = createReportGson())
```

Comment:
As for me, the creation of heavy objects like a GSON shouldn't be a default parameter to the fun.
It could lead to the creation of many doesn't reusable GSONs